config/config.py.template

```diff
@@ -589,6 +589,17 @@ if MLLIB_SPARK_VERSION >= 1.4:
     MLLIB_TESTS += [("onlinelda", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
         MLLIB_JAVA_OPTS, [ConstantOption("onlinelda")] + MLLIB_LDA_TEST_OPTS)]
 
+# TODO: tune PIC test size to run in 20-30 seconds
+MLLIB_PIC_TEST_OPTS = MLLIB_COMMON_OPTS + [
+    OptionSet("sqrt-num-points", [100], can_scale=True),
```

Comment:
Why using `sqrt`? If the node degree is fixed, the number of edges should scale linearly on number of points.